### ISSUE TITLE
Fix -80 Inventory Map

### ISSUE THREAD

**Issue by kubu4** [RobertsLab/resources#75]
The [-80C Inventory Map](https://docs.google.com/spreadsheets/d/1Qsvz3QTURlPF_hX05BQxjom3484WuMfqQ1ILl9LEljU/edit?usp=sharing) has no entries for the 5th rows and 5th columns in any of the racks. Please update when you have the chance.

**Comment by grace-ac** [comment]
Yep! A lot of those are actually just empty, so the sheet is blank to reflect that. 
But I'll go through and check to see if any racks have had things added, without updates on the spreadsheet.

**Comment by grace-ac** [comment]
Just glanced at the rack 13 section on the spreadsheet and things got messed up there, somehow. I'll fix it!

**Comment by sr320** [comment]
After subaward budget!

**Comment by kubu4** [comment]
Empty rows and columns should be shown on spreadsheet. Want full representation of entire racks.

**Comment by grace-ac** [comment]
Done! Several boxes were added/rearranged without any updates made to the spreadsheet. I think perhaps I'll put some bright tape around the protocol sign on the -80 so it is more obvious...

**Comment by sr320** [comment]
Which boxes?

**Comment by grace-ac** [comment]
Some boxes were removed, and I believe they were all things for proteomics that perhaps Emma needed.

The boxes that were added were from the middle -80 and weren't updated. And one from jose.

i did this- might be obnoxious... but it might help 
![img_4472](https://user-images.githubusercontent.com/14934314/31197438-f6ea548c-a905-11e7-9666-a388d5f9a4e3.JPG)

**Comment by sr320** [comment]
Boxes from middle -80 not updated????

https://github.com/RobertsLab/resources/issues/35#issuecomment-332332559

@laurahspencer ?

**Comment by grace-ac** [comment]
Laura's were!!!

**Comment by kubu4** [comment]
I've relocated the sign to that it doesn't cover the "No food/drink" sticker. That sticker needs to be visible to be in compliance with EH&S.

Although the information in that sign is useful, it's too wordy and most of the font is too small to spur someone to read it. I think the bigger issue here is training (or, lack thereof). I think everyone should go through the Onboarding wiki (maybe together at a lab meeting soon?). Plus, we actually <em>do</em> need most people to go through the Onboarding wiki so that we get safety training/documentation up-to-date for all lab members...

**Comment by kubu4** [comment]
Also, I'm reopening this, as the issue hasn't been resolved. The issue is that the inventory sheet doesn't match the racks. There should be a corresponding cell in the spreadsheet for every possible slot in each rack; even if the slot is currently unused.

**Comment by grace-ac** [comment]
It is up to date - I went through all the racks and updated every row. I added rows to represent empty slots in the racks.

**Comment by sr320** [comment]
I hope is was not the wrong spreadsheet 😨

**Comment by grace-ac** [comment]
Haha, I have learned from the past! I am only using the spreadsheets in the wiki!!

**Comment by grace-ac** [comment]
Which matches the one in this issue :)

**Comment by kubu4** [comment]
All good! Sorry! I was wrong! Thanks, @grace-ac!
